test/runtests.sh

```diff
@@ -16,4 +16,4 @@ if [ ! -f $ch_path ]; then
     exit 1
 fi
 
-"$test_path/runtests.py" --binary $ch_path Basics/hello.js
+"$test_path/runtests.py" -t $ch_path $test_path/Basics/hello.js
```

Comment:
This was something I was expecting. Because of path problem CI wasnt testing anything.